packages/core/src/external/athenahealth/index.ts

```diff
@@ -25,21 +25,27 @@ import {
   ProblemCreateResponse,
   problemCreateResponseSchema,
   subscriptionCreateResponseSchema,
+  VitalsCreateParams,
   VitalsCreateResponse,
   vitalsCreateResponseSchema,
 } from "@metriport/shared/interface/external/athenahealth/index";
 import axios, { AxiosInstance } from "axios";
+import dayjs from "dayjs";
 import { uniqBy } from "lodash";
 import { processAsyncError } from "../..//util/error/shared";
 import { createHivePartitionFilePath } from "../../domain/filename";
 import { fetchCodingCodeOrDisplayOrSystem } from "../../fhir-deduplication/shared";
+import { executeAsynchronously } from "../../util/concurrency";
 import { Config } from "../../util/config";
 import { SNOMED_CODE } from "../../util/constants";
 import { out } from "../../util/log";
 import { capture } from "../../util/notifications";
 import { uuidv7 } from "../../util/uuid-v7";
 import { S3Utils } from "../aws/s3";
 
+const parallelVRequests = 5;
```

Comment:
nit: typo, V between parallel and requests, or is it supposed to mean something - in which case, can we write it out in full, please?

Reply:
Fixed.